docs/docs/user/getting-started/working-with-kpops.md

```diff
@@ -0,0 +1,187 @@
+# Working with KPOps
+
+## Components
+
+This section explains the different components of KPOps, their usage and configuration.
+
+### KubernetesApp
+
+#### Usage
+
+Can be used to deploy any app in Kubernetes using helm. An example is a Redis database.
```

Comment:
I don't think that a Redis DB is a good example because that is infrastructure and not a app that processes data

Reply:
Makes sense

Could you offer a good example, please? Would, for example, redpanda console be a better one? 

From what I understand, all components (except for Connectors) inherit from KubernetesApp, but if they don't cover your needs, you  can use the `kubernetes-app` type to deploy any app with helm. 

I thought it would be good for the example to be such an app.

Reply:
Redpanda console is also part of the infrastructure. I think a better example would be a REST Service that serves Kafka Data